rpm/src/main/systemd/slipstream.service

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=SlipStream Service
 After=syslog.target network.target
-After=hsqldb.service ssclj.service
+After=hsqldb.service
```

Comment:
Are you sure this change should be part of this PR?  Also, I guess by doing so you are rolling back changes from Cal from yesterday.